tagstudio/src/core/library/alchemy/library.py

```diff
@@ -362,6 +362,8 @@ def verify_ts_folders(self, library_dir: Path) -> None:
         if not full_ts_path.exists():
             logger.info("creating library directory", dir=full_ts_path)
             full_ts_path.mkdir(parents=True, exist_ok=True)
+            # create noindex file to ignore the folder
+            (full_ts_path / TS_FOLDER_NOINDEX).touch()
```

Comment:
It seems like the purpose of this is the leverage the `noindex` system to ignore built-in folders. I would either recommend reinforcing the presence of a `.ts_noindex` file inside this folder in case it gets removed then, or just account for the folder path without the need to reinforce and then detect the presence of a `.ts_noindex` file.